Makefile

```diff
@@ -20,8 +20,20 @@ gen: manifests generate format vet ## Generate code containing DeepCopy, DeepCop
 
 .PHONY: manifests
 manifests: controller-gen ## Generate CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) crd paths="./..." output:crd:artifacts:config=charts/qdrant-operator-crds/crds
```

Comment:
these two lines got expanded into 13 lines. I would suggest to move this logic into a script under `hack` directory, keeping the makefile easier to read and making the script testable.